base/string_utils.cpp

```diff
@@ -337,6 +337,15 @@ bool EndsWith(std::string const & s1, char const * s2)
   return (s1.compare(n - m, m, s2) == 0);
 }
 
+bool EndsWith(std::string const & s1, std::string_view s2)
```

Comment:
Is there a unit-test for this method?

Reply:
Added tests for EndsWith(string, string_view)